test/src/core/SceneRunnerCSharpSceneTest.cs

```diff
@@ -71,6 +71,16 @@ public void LoadSceneBinaryFormat()
             .IsNotNull();
     }
 
+
+    [TestCase]
+    public void LoadSceneNode()
+    {
+        using var runner = ISceneRunner.Load(new Node(), true);
```

Comment:
This is a bad example, the test should use a real scene with using a custom initializer method before load by the scene runner.